### PR TITLE
PLAT-48: Restructure the application directories

### PR DESCRIPTION
**Pull request by dondrzzy**
## JIRA TICKET NAME
[PLAT-48: Restructure the application directories](https://issues.openmrs.org/browse/PLAT-48)

## Description of what I changed

- Moved Login component from `/js/components/` to `/js/components/login/`.
- Moved LoginForm component from `/js/components/presentational/` to `/js/components/login/`

## Issue I worked on
[PLAT-48](https://issues.openmrs.org/browse/PLAT-48)

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Review by IAMOTZ — APPROVED** [review]

**Review by IAMOTZ — APPROVED** [review]
Nice one @dondrzzy.

**Review by dotNesh — APPROVED** [review]
LGTM!

**Review by alexmochu — APPROVED** [review]
Good job.

File: app/js/components/header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import logo from '../../../img/openmrs-with-title-small.png';
+
+export default () => (
```
Comment (dkayiwa) [inline app/js/components/header/Header.jsx]: Is this restructuring a directory or adding a totally new class?

Comment (dondrzzy) [reply app/js/components/header/Header.jsx]: That is not supposed to be in this PR. I have removed it.

**Review by Efosaok — APPROVED** [review]
LGTM

**Pull request merged**